pipenv/cli.py

```diff
@@ -37,7 +37,7 @@
 click_completion.init()
 
 # No color mode, for unfun people.
-if 'PIPENV_COLORBLIND' in os.environ:
+if os.environ.get('PIPENV_COLORBLIND') == '1':
```

Comment:
While this it's possible the user could set the value to these, there's a whole host of options that *could* be used if we open it up to that. What about y, n, TrUE, 0x01? It's not really worth the time trying to guess what people *might* use here. `1` is the standard "yes" flag and should be sufficient.